src/formatDate.js

```diff
@@ -8,7 +8,55 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newFormat = [];
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+  const numbers = date.split(separatorFrom);
```

Comment:
The `split` method assumes that the separator is correct and present in the date string. Consider adding error handling to manage cases where the separator is incorrect or missing.